tsrc/cli/main.py

```diff
@@ -130,11 +130,11 @@ def main(args: ArgsList = None) -> None:
     push_parser.add_argument("-t", "--target", dest="target_branch")
     push_parser.add_argument("push_spec", nargs="?")
     push_parser.add_argument("-a", "--assignee", dest="assignee")
+    push_parser.add_argument("-r", "--reviewer", "--approver", dest="reviewers", action="append",
```

Comment:
Instead of creating a new `--approver` option in the gitlab group, I just moved the `--reviewer` option to the push_parser group.

I did this because I wanted a short option available.

I could also have added an option like `-p, --approver`, but not sure if it's worth it.